.github/workflows/lint.yml

```
@@ -46,13 +46,16 @@ jobs:
           (! git grep -Il '' -- . ':(exclude)**/contrib/**' ':(exclude)third_party' ':(exclude)**.expect' ':(exclude)tools/clang_format_hash' | tools/trailing_newlines.py || (echo "The above files do not have correct trailing newlines; please normalize them"; false))
       - name: Ensure no trailing spaces
         run: |
-          (! git grep -I -no ' $' -- . ':(exclude)**/contrib/**' ':(exclude)third_party' || (echo "The above files have trailing spaces; please remove them"; false))
+          (! git grep -In ' $' -- . ':(exclude)**/contrib/**' ':(exclude)third_party' || (echo "The above lines have trailing spaces; please remove them"; false))
```

```suggestion
          (! git grep -In '[[:blank:]]$' -- . ':(exclude)**/contrib/**' ':(exclude)third_party' || (echo "The above lines have trailing spaces; please remove them"; false))
```

What about using `[:blank:]`?

---

That could work! Do you mind if I try that in a followup rather than in this PR?

---

Eh, actually that only matches spaces and tabs, so it might not be a better alternative. 

I know [:space:] does try to match all the whitespace, but then it overdoes it with new lines and carriage returns.